nodes/scheduling/nodes-scheduler-node-names.adoc

```diff
@@ -27,5 +27,4 @@ include::modules/nodes-scheduler-node-names-configuring.adoc[leveloffset=+1]
 [id="nodes-scheduler-node-names-addtl-resources_{context}"]
 == Additional resources
 
-For information about changing node labels, see xref:../../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes].
-
+For information about changing node labels, see xref:../nodes/nodes-nodes-working.adoc#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes].
```

Comment:
Same comment as above file. There are more `../nodes/nodes` -> `nodes` in the below files if you can check them out to see if they need to be reverted - I'm not going to point out any more instances